force-app/main/default/lwc/githubViewerListOfCommits/githubViewerListOfCommits.js

```diff
@@ -0,0 +1,75 @@
+import { LightningElement, api } from 'lwc';
+import fetchCommits from '@salesforce/apex/GitHubCommitsHandler.fetchCommits';
+
+export default class CommitsList extends LightningElement {
+    commits;
+    selectOptions = [];
+    commitsData = [];
+    choosenCommit;
+
+    connectedCallback() {
```

Comment:
[MAJOR/DOUBT]
To będzie uderzało do zewnętrznego systemu za każdym razem jak się wróci do listy commitów? Zadbajmy o limity API calls. Niech to się zaciąga tylko raz w czasie życia komponentu